lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -113,6 +114,33 @@ describe('modules/manager/npm/extract/index', () => {
       });
     });
 
+    it('returns an array of dependencies when is a yaml version too', async () => {
+      const res = await npmExtract.extractPackageFile(
+        input01YamlContent,
+        'package.yaml',
+        defaultExtractConfig,
+      );
+      expect(res).toMatchSnapshot({
```

Comment:
Use `toMatchObject` instead. We prefer not to add new snapshots as it affects performance